test/models/domain_test.rb

```diff
@@ -25,14 +25,4 @@ def setup
     assert_not @domain.valid?
     assert_includes @domain.errors[:fqdn], "can't be blank"
   end
-
```

Comment:
why did these tests get removed?

Reply:
the `groups_array` function was removed (Domain.groups is now serialized as array automatically)